px-datetime-entry-cell.html

```diff
@@ -246,12 +246,13 @@
       }
       ctx.font = fontSize + " " + fontFamily;
       if(this.$$('.datetime-entry-input').value) {
-        length = ctx.measureText(this.$$('.datetime-entry-input').value.toUpperCase()).width + 1;
+        length = ctx.measureText(this.$$('.datetime-entry-input').value.toUpperCase()).width;
       }
       else {
         length = ctx.measureText(this.$$('.datetime-entry-input').placeholder).width;
       }
-      length = Math.ceil(length);
+      // input doesn't have enough spacing on Edge and Firefox for value & placeholder text
+      length = Math.ceil(length) + 2;
```

Comment:
the width modification for the placeholder text should be on line 252, since that spacing only affects the placeholder. please update accordingly.

Update the comment to reflect that the extra width is needed for the placeholder ONLY since the text in the placeholder is styled italicized, which takes up more horizontal space.

Reply:
Updated placeholder spacing on line 252 and comment accordingly.